lib/types/enums.ts

```diff
@@ -7,3 +7,13 @@ export enum OrderingDirection {
   DESC = 'DESC',
   ASC = 'ASC',
 }
+
+export enum CoinType {
```

Comment:
If we keep this it should be `CurrencyType` to stay consistent, but I'm concerned that hardcoding them here conflicts with the existence of our `currencies` table which is supposed to track the supported currencies.

Instead I think we should just refer to currencies by their tickers as strings (which is the primary key for the currencies table) and not as an enum.

Reply:
Change to CurrencyType.
Added TODO for currencies